DOCS/man/input.rst

```diff
@@ -447,6 +447,12 @@ Property Manipulation
 Playlist Manipulation
 ~~~~~~~~~~~~~~~~~~~~~
 
+``screenshot-to-clipboard [<flags>]``
```

Comment:
the documentation for this should be under `Screenshot Commands` not `Playlist Manipulation`.

also please correct me if i am wrong. this is raw bitmap data with pass here or at least not compressed data (png, jpg, whatever is configured with out screenshot options)? so shouldn't this be `screenshot-raw-to-clipboard`?

eg i would expect `screenshot-to-clipboard` to write a screenshot to the clipboard the in the same format `screenshot` writes a screenshot to the file system. so it should be affected by 'all' the `screenshot-*` options.

Reply:
`screenshot-to-file` also ignores the format option, though?

Reply:
i guess the documentation mentions the `screenshot-to-file` ignores at least the format option. my expectations were a bit betrayed then.

> Take a screenshot and save it to a given file. The format of the file will be guessed by the extension (and --screenshot-format is ignored - the behaviour when the extension is missing or unknown is arbitrary).

though thinking a bit about the future. what if we want to actually add `screenshot-to-clipboard` command that wants to add the aforementioned case (eg adding a screenshot as png, jpeg, etc)? would we add a new flag to this command? i am just wondering and trying to prevent a possible 'mess'.

though if we do what @na-na-hi proposed we don't need to think further about this case.

Reply:
There's a conceptual difference between an "image on the clipboard" and a "file on the clipboard", and generally copying ephemeral image data does not result in you having a file on the clipboard. On macOS, images on the clipboard are commonly represented as PNG or TIFF data (sometimes both), but there is no eg JPEG clipboard format. There's also no concept of arbitrarily ephemeral file contents on the clipboard (only a file *path*). The same is true on Windows afaik, except that the image format is BMP instead of PNG/TIFF. I have no idea how this works on linux, apart from that apparently on X it's common for apps to expose several different types for a copied image.

Images copied out of mpv on this branch are PNG data (but that's an implementation detail, and it basically never matters to a user).